### ISSUE TITLE
[BUG] EMAIL_LETS_ENCRYPT regex does not support Sub-addressing

### ISSUE THREAD

**Issue by Crazy3lf** [bunkerity/bunkerweb#698]
### What happened?

 The current regex does not allow using email [sub-address](https://en.wikipedia.org/wiki/Email_address#Sub-addressing) for `EMAIL_LETS_ENCRYPT`. Subaddress is accepted by Let's Encrypt because one was use in the [example of unsubscribing](https://letsencrypt.org/docs/expiration-emails/#unsubscribing):
> However, you can change the email address on your account, which effectively re-subscribes you. Many common email services treat `yourname+1@example.com` the same as `yourname@example.com`. So if you update your email address to `yourname+1@example.com`, you can start getting expiry mail again. With Certbot, use:
> 
> `certbot update_account --email yourname+1@example.com`

Adding this support should be just a quick fix to the regex.

### How to reproduce?

Use a subaddress such as `abcd+letsEncrypt1@dmail.com` for `EMAIL_LETS_ENCRYPT` env setting.

### Configuration file(s) (yaml or .env)

```YAML
version: '3.8'
services:
  myapp1:
    image: tutum/hello-world
    networks:
      bw-services:
    deploy:
      labels:
        - "bunkerweb.SERVER_NAME=www.helloworldt.com"
        - "bunkerweb.AUTO_LETS_ENCRYPT=yes"
        - "bunkerweb.EMAIL_LETS_ENCRYPT=abcd+letsEncrypt1@dmail.com"
        - "bunkerweb.USE_REVERSE_PROXY=yes"
        - "bunkerweb.REVERSE_PROXY_URL=/"
        - "bunkerweb.REVERSE_PROXY_HOST=http://myapp1"

networks:
  net-bw-services:
    external: true
    name: bw-services
```

### Relevant log output

```shell
Scheduler: 
[2023-10-16 09:05:55] - GENERATOR - ⚠️  - Ignoring variable www.helloworldt.com_EMAIL_LETS_ENCRYPT : value abcd+letsEncrypt1@dmail.com doesn't match regex ^([\w.\-]+@[a-zA-Z0-9.\-]+\.[a-z]{2,})?$
```

### BunkerWeb version

1.5.2

### What integration are you using?

Swarm

### Removed private data

- [X] I have removed all private data from the configuration file and the logs

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by TheophileDiot** [comment]
Hi @Crazy3lf, thank you for opening this issue. When I created the regex I was maybe a bit to harsh, we indeed need to lose some of them.

**Comment by fl0ppy-d1sk** [comment]
Hello @Crazy3lf,

It should be fixed in the last 1.5.3 release. Don't hesitate to open a new issue if needed.
